src/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  font-family: inherit;
```

Comment:
your .card doesn't have major rules about fonts, and why did you put here inherit property?